google/cloud/storage/tests/object_file_multi_threaded_test.cc

```diff
@@ -33,15 +31,21 @@ namespace {
 
 using ::testing::HasSubstr;
 
-// Initialized in main() below.
-char const* flag_bucket_name;
-int flag_object_count;
-
 auto const kObjectSize = 16 * 1024;
 
 class ObjectFileMultiThreadedTest
     : public google::cloud::storage::testing::StorageIntegrationTest {
  protected:
+  void SetUp() override {
+    bucket_name_ = google::cloud::internal::GetEnv(
+                       "GOOGLE_CLOUD_CPP_STORAGE_TEST_BUCKET_NAME")
+                       .value_or("");
+    ASSERT_FALSE(bucket_name_.empty());
+    auto object_count = google::cloud::internal::GetEnv(
+        "GOOGLE_CLOUD_CPP_STORAGE_TEST_OBJECT_COUNT");
+    if (object_count) std::stoi(*object_count);
```

Comment:
Should we be assigning to `object_count_` here?

Reply:
Fixed, thanks.